src/lib/icons/search_16px.svg

```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" viewBox="0 0 15 14">
+    <g fill="none" fill-rule="evenodd">
```

Comment:
Is this group needed?

Reply:
Yeah, a black background appears behind the search icon without it